functions/__sf_section_ruby.fish

```diff
@@ -12,7 +12,7 @@ function __sf_section_ruby -d "Show current version of Ruby"
 	__sf_util_set_default SPACEFISH_RUBY_SHOW true
 	__sf_util_set_default SPACEFISH_RUBY_PREFIX $SPACEFISH_PROMPT_DEFAULT_PREFIX
 	__sf_util_set_default SPACEFISH_RUBY_SUFFIX $SPACEFISH_PROMPT_DEFAULT_SUFFIX
-	__sf_util_set_default SPACEFISH_RUBY_SYMBOL "💎 "
+	__sf_util_set_default SPACEFISH_RUBY_SYMBOL "💎  "
```

Comment:
Is there a reason this extra space was added?

Reply:
Sure, maybe it's just my font, but this looked super weird with a monospace font. I'm fine removing this, just thought it looked better. Here's what it looks like without the space:

<img width="389" alt="screen shot 2018-07-06 at 12 41 35 am" src="https://user-images.githubusercontent.com/2652762/42360212-5d5b1d78-80b5-11e8-961a-9cf1b2a2896b.png">

Reply:
Yep I agree! However spacefish's [philosophy](https://github.com/matchai/spacefish/blob/master/CONTRIBUTING.md#philosophy) is to follow [spaceship's standards](https://github.com/denysdovhan/spaceship-prompt/blob/master/sections/ruby.zsh#L14) as much as possible. So it would be up to the individual user to decide if they want to change the default one space.

Reply:
Oh man I totally misunderstood what was going on, sounds great! I'll remove this.

Reply:
If you're encountering this in iTerm, take a look at this setting: https://github.com/matchai/spacefish/blob/master/docs/Troubleshooting.md#why-do-my-section-icons-overlap-each-other

Reply:
This is great, thanks @matchai !

Reply:
Turns out I actually can't use this setting @matchai :( I'm using powerline fonts and it messes with those characters